react/LocationCountry.tsx

```diff
@@ -107,7 +109,7 @@ const LocationCountry: React.FC<LocationCountryProps> = ({
     <>
```

Comment:
you can change this fragment to a div and add the className here, else you'll only hide the component's label

Reply:
I don't know what I was thinking. Done!